chapter_09_docker.asciidoc

```diff
@@ -276,6 +278,8 @@ How will containerizing our software help with the danger areas?
 
 * Later, when we deploy our containers to a staging server,
   we can run the FTs against that too.
+// XUAN: "FTs" = "functional tests"?
+// XUAN: This is the first time this chapter mentions "staging", which seems a bit out of the blue. Can you elaborate on what problems does having a staging server solve for the initial list of danger areas of deployment?
```

Comment:
thanks, yes!  i think maybe a more explicit discussion of the incremental approach, docker first, run tests against that, staging second, run tests against that, prod third...